README.md

```diff
@@ -597,6 +599,19 @@ class B {
 // Instead of the above, try something like this:
 expectTypeOf(B).instance.toEqualTypeOf<{b: string; foo: () => void}>()
 ```
+
+Overloads limitation: overloaded functions which include an overload looking like `(...args: unknown[]) => unknown` will excluded `unknown[]` from `.parameters` and exclude `unknown` from `.returns`. This overload just allows any input and returns unknown output anyway though.:
```

Comment:
```suggestion
Overloads limitation: Overloaded functions that include an overload resembling `(...args: unknown[]) => unknown` will exclude `unknown[]` from `.parameters` and `unknown` from `.returns`. This overload, however, allows any input and returns an unknown output anyway:
```

Reply:
re-jigged this a little since you commented but done.